web/.angular-cli.json

```diff
@@ -22,10 +22,16 @@
       "testTsconfig": "tsconfig.spec.json",
       "prefix": "app",
       "serviceWorker": true,
+      "scripts": [
+        "../node_modules/jquery/dist/jquery.min.js",
```

Comment:
Are you sure it is the correct way to add dependencies?

Reply:
Quite, otherwise it attempts to look for these additional chunk assets in the "src" directory. That's why I added "../".

Reply:
I mean why we add the scripts manually? Is it not enough to add it in the package.json?

Reply:
Not enough endeed. $ng serve throws an error like "couldn't find /path/node_modules/jquery.min.js".
I've also seen them doing it manualy in the online tutorials: for bootstrap they added it manualy to override the default styles and so on.

Reply:
It's a pretty strange situation. This change looks like a huck. When you add npm package you shouldn't add it anywhere. I think we need to investigate the situation and find out why you get this error.

Reply:
Maybe the thing is that in order to use bootstrap/jquery we have to add it either via CDN or via NPM. In case of CDN - we need to insert links into index.html to reach bootstrap libraries. In case of NPM - it installs packages(libraries) into node_modules dir and adds dependencies to package.json. But we still have to write paths to those libraries - that's why we add them to .angular-cli.json (or insert links into index.html again)

Reply:
Ok, I think you right. :)